flask_resources/resources.py

```diff
@@ -79,6 +81,9 @@ def as_blueprint(self, name, **bp_kwargs):
         for rule in self.create_url_rules(name):
             blueprint.add_url_rule(**rule)
 
+        # register error handlers
+        self.create_error_handlers(blueprint)
```

Comment:
The pattern should be the same for both ``create_url_rules`` and ``create_error_handlers``. I.e. either they return an iterable of rules or error handlers, or a blueprint is passed in.